### ISSUE TITLE
JAWS issue with ghost icon button repeating description twice when disabled

### ISSUE THREAD

**Issue by drjoho** [carbon-design-system/carbon#8501]
## What package(s) are you using?

- [ ] `carbon-components`
- [x ] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.

If you use JAWS in read mode (not navigate mode) when it gets to the ghost icon button the description is repeated twice when the button is disabled. Works fine when active.

> Is this issue related to a specific component?

Carbon ghost icon button.

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?
Chrome
Firefox

> What version of the Carbon Design System are you using?

"@carbon/themes": "^10.31.0",
"carbon-components": "^10.10.1",
"carbon-components-react": "^7.10.1",
"carbon-icons": "^7.0.7",

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

## Steps to reproduce the issue

Start up the IBM Equal Accessibility Checker. Don't scan. Use JAWS in read mode. When you reach the disabled reports button it will read the description twice.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/react/examples/codesandbox

## Additional information

- Screenshots or code

It offending button is the reports button under the Bee. Works fine when active. Issue is when disabled.

![image](https://user-images.githubusercontent.com/64442976/116094220-f4e10b00-a66c-11eb-8a7b-a12ae885e7cd.png)

- Notes

**Comment by dakahn** [comment]
Seems like you might be a bit behind -- is an update possible? 

See if you can recreate this bug in our Storybook using latest carbon-react. I'm unable to 👍🏽
